services/orchestrator/src/main/java/org/eclipse/osc/orchestrator/plugins/huaweicloud/HuaweiCloudPlugin.java

```diff
@@ -0,0 +1,48 @@
+package org.eclipse.osc.orchestrator;
+
+import lombok.extern.java.Log;
+import org.apache.karaf.minho.boot.spi.Service;
+import org.eclipse.osc.services.ocl.loader.Ocl;
+
+@Log
+public class HuaweiCloudPlugin implements OrchestratorPlugin, Service {
```

Comment:
`HuaweiCloudPlugin` should be in its own module, not directly in the `Orchestrator`.

I think it's better to create `services/huaweicloud`. This module will have `orchestrator` as dependency and implements `HuaweiCloudPlugin`.

I'm doing the same with `services/openstack`.

Like this, we have a complete extensible mechanism where users can provide their own plugins.

It's in the runtime that we define the plugins we want (in the assembly).
We can imagine to have several runtime (like `osc-huaweicloud`, `osc-opentstack`, `osc-aws`, ...).